packages/react-app/src/components/Balance.jsx

```diff
@@ -33,6 +33,12 @@ const { utils } = require("ethers");
 export default function Balance(props) {
   const [dollarMode, setDollarMode] = useState(true);
 
+  let RPC_POLL_TIME = LOCAL_RPC_POLL_TIME;
+
+  if (props.provider && props?.provider._network.chainId === 31337) {
+    RPC_POLL_TIME = 0;
+  }
+
```

Comment:
Since we repeat a similar logic in `App.jsx` and `FauceHint.jsx`.. how do you feel about creating a util for it?

E.g. Create a `rpc.js` file inside the `helpers` folder that exports a `getRpcPollTime(provider)` that handle the logic.

Reply:
Yes completely agreed,  will for sure keep this in mind next time 🙌